src/App.jsx

```diff
@@ -14,70 +16,76 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [product, setProduct] = useState('Jam');
+  const value = '-';
```

Comment:
The variable 'value' is declared but its value is never read. Consider removing it if it's not used.